crit/mempages.go

```diff
@@ -0,0 +1,134 @@
+package crit
+
+import (
+	"bytes"
+	"fmt"
+	"os"
+	"path/filepath"
+
+	"github.com/checkpoint-restore/go-criu/v6/crit/images/pagemap"
+)
+
+const pageSize = 4096
```

Comment:
Please use `os.GetPageSize()` as it can be a different size.

Reply:
Wouldn't it be better to have `pageSize` as a field of `MemoryReader` so that users of the package can set the value themselves when creating new instance of `MemoryReader` ? `os.Getpagesize()` will still be the default value.